app/src/main/java/com/example/bigowlapp/activity/LoginPageActivity.java

```diff
@@ -54,10 +58,10 @@ protected void initialize() {
             btnSignIn.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
+                    progressBar.setVisibility(View.VISIBLE);
```

Comment:
You should instead put this right above the db call (line 72), because if for example the email is empty, I get an infinite loading bar now.

Reply:
Nice catch.